afterburners/client.lua

```diff
@@ -26,52 +23,46 @@ else
 end
 	
 
-Citizen.CreateThread(function()
+CreateThread(function()
 	while true do
-		Citizen.Wait(50)
-		
+		Wait(50)
 		local handle, veh = FindFirstVehicle()
 		local finished = false -- FindNextPed will turn the first variable to false when it fails to find another ped in the index
 		repeat
 			model = GetEntityModel(veh)
-			if IsThisModelAPlane(model) and GetVehiclePedIsIn( PlayerPedId(), false ) == veh then
+			if IsThisModelAPlane(model) and GetVehiclePedIsIn(PlayerPedId(), false) == veh then
 				tryAfterburner(veh)
 			end
 			local finished, veh = FindNextVehicle(handle) -- first param returns true while entities are found
 			model = GetEntityModel(veh)
-			if IsThisModelAPlane(model) and GetVehiclePedIsIn( PlayerPedId(), false ) == veh then
+			if IsThisModelAPlane(model) and GetVehiclePedIsIn(PlayerPedId(), false) == veh then
 				tryAfterburner(veh)
 			end
 			until not finished
 			EndFindVehicle(handle)
 			
-			if GetVehiclePedIsIn( PlayerPedId(), false ) and IsThisModelAPlane(GetEntityModel(GetVehiclePedIsIn(PlayerPedId(),false) ) ) then
-				tryAfterburner( GetVehiclePedIsIn(PlayerPedId(),false))
-			end
-			
-			
-				
+			if GetVehiclePedIsIn(PlayerPedId(), false) and IsThisModelAPlane(GetEntityModel(GetVehiclePedIsIn(PlayerPedId(),false))) then
```

Comment:
why remove citizen calls and not optimize `PlayerPedId` usage?

Reply:
What do you mean?

Reply:
You are removing the usage of the Citizen prefix, however, completely ignore that PlayerPedId() is called over 5 times every frame, which would be _way_ more intensive, your PR is amateurish at best and at worst, completely misinformed.

Reply:
So if you know how to do it why not do it? Just update the fxmanifest that i put and do it?

Reply:
this is 5 year old code, why would i randomly update it for no reason whatsoever, when it already works fine? all of your changes (less for the fxmanifest) are purely cosmetic and do not actually improve anything, code style changes like this have no place in a PR without previously even asking if its fine to do so, as there is very likely a reason why every Citizen call is prefixed with it.

You are free to run your own fork, however, making random codestyle changes and then getting sassy when i decline them just proves your ignorance on the topic.

Reply:
Why would i ask permission to change a few lines of code xD its a pull request deny it if you want just me trying to help with your old outdated code then you complaining at me for doing it, my man over here is mad

Reply:
you "trying to help with my outdated code" was actually _broken_ before i reviewed the PR, maybe instead of being a script kiddie and making minor and frankly, irrelevant changes, actually look into what the code does and optimize it from there, look at your PRs changes, the only relevant change you did was change from __resource.lua to a (broken) fxmanifest.

You are wasting my time by having me review this garbage, i'm not mad, i'm annoyed and dissapointed that you dont see how this is annoying for a maintainer.

Reply:
Wouldnt call yourself a maintainer if you dont maintain it pal, Made a mistake cause was doing it for a mate who was using your outdated code. Not really a script kiddie tbh but its funny seeing you getting mad over your own code being old and outdated